playwright-e2e/playwright.config.ts

```diff
@@ -16,8 +16,8 @@ const testConfig: PlaywrightTestConfig = {
   globalSetup: require.resolve('./fixtures/global-setup'),
   testDir: __dirname,
   testMatch: '**/*.spec.ts',
-  /* Maximum timeout per test. Each test should be short and takes less than 4 min to run */
-  timeout: 240 * 1000,
+  /* Maximum timeout per test. Each test should be short and takes less than 5 min to run */
+  timeout: 5 * 60 * 1000,
```

Comment:
To allow more time if running all tests on localhost from home (slower network speed).